lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -207,6 +209,12 @@ def self.is_firebasecmd_supported?(cmd)
 
         true
       end
+      def self.post_notes(app_id, release_id, release_notes)
+       # payload = {releaseNotes: {releaseNotes: release_notes}}
+        connection.post("#{PATH}#{app_id}/releases/#{release_id}/notes", {releaseNotes: {releaseNotes: release_notes}}.to_json) do |request|
+          request.headers["Authorization"] = "Bearer " + auth_token
+        end
```

Comment:
Also I will be updating this with a message that the notes have been posted soon.